src/api/profile/content-types/profile/schema.json

```diff
@@ -44,6 +44,18 @@
       "config": {
         "provider": "azure-storage"
       }
+    },
+    "location": {
+      "type": "string"
+    },
+    "role": {
```

Comment:
Is this the user's role (developer, designer, etc) inside Dev Launchers?

Reply:
Hey @chungthuang, this will describe how they want to do or are doing - a designer, developer etc.. 
Basically how they want to be viewed as.  I've attached a screenshot of it

<img width="571" alt="image" src="https://github.com/user-attachments/assets/2a562c3d-f66f-414f-b350-f6a7879c1f7d">

Reply:
So this looks like we have category of roles (developer, UX/UI, other) which is similar to [roleCategory in opportunity collection](https://github.com/dev-launchers/strapiv4/blob/main/src/api/opportunity/content-types/opportunity/schema.json#L59). I think this is a good use case for components, so we share the same role category across our collections.

Reply:
Something like https://github.com/dev-launchers/strapiv4/commit/64f20421b0880f413b71365fe5ad5232fd6295d9.